components/tooltip/nz-tooltip.spec.ts

```diff
@@ -154,6 +156,20 @@ describe('NzTooltip', () => {
       expect(overlayContainerElement.textContent).not.toContain(title);
     }));
 
+    it('should not show tooltip with nullable trigger', fakeAsync(() => {
```

Comment:
You can remove tests in popover and popconfirm. Since they share code from tooltip/base.